app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -130,7 +130,7 @@ form {
 
     &:focus {
       border: $border-width solid #99a2aa;
-      @include shadow(0,0,4px,#99a2aa);
+      box-shadow: 0 0 4px #99a2aa;
```

Comment:
Color literals like `#99a2aa` should only be used in variable declarations; they should be referred to via variable everywhere else.